files/swap.sh

```diff
@@ -28,13 +29,19 @@ then
   fi
 fi
 
+if [ $MKSWAP_MIN -gt $SWAP_TO_CREATE ]; then
+  SWAP_TO_CREATE=$MKSWAP_MIN
+fi
+
+
 if [ "$SWAP" == "none" ]
 then
   exit 0
 else
-  TMP=`mktemp`
+  TMP='/var/oracle-xe.swapfile'
```

Comment:
Why did you change this to a fixed name instead of the dynamic name returned by `mktemp`?